Dockerfile

```diff
@@ -15,17 +25,27 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-RUN pip install --upgrade pip
-RUN conda update setuptools
+RUN chown -R ${NB_UID} /opt/conda/
+
+COPY requirements.txt ${HOME}/
+
+USER ${NB_USER}
+
+RUN pip install --upgrade pip && \
+      pip install -r ${HOME}/requirements.txt && \
+      conda update setuptools && \
+      conda install jupyter notebook scikit-learn
```

Comment:
Can you align lines 35-37 (it seems to have 5 spaces instead of 4 used all over the file).

Reply:
Solved.

This was a problem in my editorconfig.